SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -73,7 +73,7 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         val departureDateTime = processTimeForDisplay(journeyDetails.departureTime)
         val arrivalDateTime = processTimeForDisplay(journeyDetails.arrivalTime)
         val journeyTime: TimeSpan = arrivalDateTime - departureDateTime
-        val journeyTimeMinutes: String = "${journeyTime.minutes}m"
+        val journeyTimeMinutes: String = "${journeyTime.minutes.toInt()} min"
```

Comment:
If you want to use  just m in the display, but want min in the voiceover you could look at using the `accessibilityLabel` trait 🙂